items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -121,6 +121,31 @@ class SierraItemUpdater(sierraSource: SierraSource)(
             case None            => item
           }
       }
-    } yield updatedItems.toSeq
+
+      updatedItemsWithAvailableDates = updatedItems.map {
+        case (item) =>
+          val displayAccessConditionOps = new DisplayAccessConditionOps(
+            item.locations.head.accessConditions.head
+          )
+          displayAccessConditionOps.isRequestable match {
+            case true =>
+              item.copy(
+                availableDates = Some(
+                  List(
+                    AvailabilitySlot(
+                      ZonedDateTime.parse(
+                        "2024-02-29T13:32:44.943107Z[Europe/London]"
+                      ),
+                      ZonedDateTime
+                        .parse("2024-02-29T13:32:44.943107Z[Europe/London]")
+                        .plusWeeks(2)
+                    )
+                  )
+                )
+              )
+            case false => item
+          }
+      }
+    } yield updatedItemsWithAvailableDates.toSeq
```

Comment:
In anticipation of the next stage of work, I would suggest splitting out the addition of available dates into another class or something - so that no changes would be required here when we use "real" dates